src/Storages/ObjectStorageQueue/StorageObjectStorageQueue.cpp

```diff
@@ -300,17 +301,12 @@ void StorageObjectStorageQueue::shutdown(bool is_drop)
             tryLogCurrentException(log);
         }
 
```

Comment:
I assume we can remove `unregister` call above :thinking:

Reply:
`ObjectStorageQueueMetadataFactory::instance().remove` calls unregister inside but with `active = false`.

Reply:
yes, those are two different registry types